src/main/java/graphql/execution/FieldCollectorParameters.java

```diff
@@ -26,15 +26,15 @@ public GraphQLSchema getGraphQLSchema() {
         return variables;
     }
 
-    public GraphQLObjectType getObjectType() {
-        return objectType;
+    public GraphQLFieldsContainer getFieldsContainer() {
+        return fieldsContainer;
     }
```

Comment:
Not sure about this change: why is this better? looking at the code in `checkTypeCondition` it doesn't change anything but makes it more complicated: we are only interested in objects anyway.